package.json

```diff
@@ -109,10 +109,10 @@
     "@angular-devkit/build-angular": "~0.6.0",
     "@angular-devkit/core": "^0.6.5",
     "@angular-devkit/schematics": "^0.6.5",
+    "@angular/cdk": "6.0.0",
```

Comment:
Is it important change?

Reply:
it is automatic by npm, so it will appear now or in future